charts/rancher-logging/rancher-logging/3.14.0/values.yaml

```diff
@@ -147,10 +147,10 @@ images:
     tag: v0.4.0
   fluentbit:
     repository: rancher/mirrored-fluent-fluent-bit
-    tag: 1.8.7
+    tag: 1.6.10
```

Comment:
This seems like quite a step backward, did nothing in the 1.8.x series work? What actually is the issue? Is it this https://github.com/fluent/fluent-bit/issues/4050 ?

Reply:
I tried a few different 1.8.x version and they all have the same issue.  I don't see it as a step backwards, it is more of opting out of an upgrade.  1.6.10 is what we are currently using in rancher logging.

Reply:
Looks like it might be fixed in the next release https://github.com/fluent/fluent-bit/issues/4050#issuecomment-934803218 if we can't wait for that can we make sure there's a followup issue to upgrade when we can?

Reply:
https://github.com/rancher/rancher/issues/35058